tex/chProofs.tex

```diff
@@ -4,6 +4,7 @@
 Set Implicit Arguments.
 Unset Strict Implicit.
 Unset Printing Implicit Defensive.
+% [DG] This looks outdated -- at least it doesn't compile.
```

Comment:
Hi @darijgr! We are aware that the way we handle code at the moment is awful. But these 3 lines should make sense however. Could you say more about the problem you had here?